.github/workflows/container-test-deploy.yml

```diff
@@ -42,6 +42,8 @@ jobs:
           context: ./all-in-one
           load: true
           tags: ${{ env.IMAGE_NAME }}
+          build-args: |
+            OSCAL_REST_SERVICE_GIT_BRANCH=feature/combined-test
```

Comment:
This must be reverted (and the related PRs must be merged) before this can be merged.